scripts/solpb.sh

```diff
@@ -6,10 +6,14 @@ if [ -z "$SOLPB_DIR" ]; then
     exit 1
 fi
 
+GEN_DIR="$(pwd)/generated"
+mkdir $(pwd)/generated
```

Comment:
```suggestion
mkdir "$GEN_DIR"
```

Reply:
@siburu I've made a small suggestion. (However, if you try to apply it using the github's feature, I guess the commit may not contain Signed-off.)

Reply:
@bluele Thank you! I've just fixed it.